test/helpers/setup-helper.js

```diff
@@ -89,10 +91,35 @@ window.document.body.appendChild(popoverContent);
 // eslint-disable-next-line no-shadow
 const { default: fetch, Headers, Request, Response } = require('node-fetch');
 
-Object.assign(window, { fetch, Headers, Request, Response });
+const shimmedFetch = (url, ...args) => {
+  return fetch(url, ...args).catch((error) => {
+    if (error.message === 'Only absolute URLs are supported') {
+      const regex = /_locales\/([^/]+)\/messages\.json/gu;
```

Comment:
Do we need to limit it to the messages, or could we keep it general to any relative URL?

Reply:
We do not... but for each available static asset, the path for it will depend on the static assets generated/placed during build ([here](https://github.com/MetaMask/metamask-extension/blob/develop/development/build/static.js#L111)). For that reason I would prefer to keep this targeted, and eventually if there's a new case to get a image or something like that, we can update the shim to take those other static assets into account